contracts/protocol/pool/PoolStorage.sol

```diff
@@ -31,4 +31,7 @@ contract PoolStorage {
 
     // Maximum allowed number of atomic tokens per user
     uint24 internal _maxAtomicTokensAllowed;
+
+    // Auction recovery health factor
+    uint256 internal _auctionRecoveryHealthFactor;
```

Comment:
Does this require unit256? Or can we stack it with the previous state var?

Reply:
also this is. global number for all reserves. are we sure we want to keep it this way or make it more dynamic per-reseerve?